src/test/java/org/broadinstitute/hellbender/tools/dataflow/pipelines/CountBasesDataflowIntegrationTest.java

```diff
@@ -0,0 +1,64 @@
+package org.broadinstitute.hellbender.tools.dataflow.pipelines;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class CountBasesDataflowIntegrationTest extends CommandLineProgramTest{
```

Comment:
I think we need to have a unified way to test dataflow and non-dataflow tools. Otherwise integration tests will get out synch very fast.